tools/Az.Tools.Installer/exports/Update-AzModule.ps1

```diff
@@ -91,7 +91,12 @@ function Update-AzModule {
             $findModuleParams.Add('Repository', $Repository)
         }
         $modulesToUpdate = Get-AzModuleFromRemote @findModuleParams
-        $Repository = $modulesToUpdate.Repository | Select-Object -First 1
+        if ($modulesToUpdate) {
+            $Repository = $modulesToUpdate.Repository | Select-Object -First 1
+        }
+        else {
+            $Repository = (Get-PSRepository | Select-Object -Property Name)
```

Comment:
Get-PSRepository may return more than 1 result. This like also needs "-First 1"?